### PR TITLE
Adjust styles to the new mock-ups

### PR DESCRIPTION
**Pull request by kovtun-d**

### CODE REVIEW

**Comment by RomainLofaso** [comment]
I observe that we use many format for the fonts: eot, ttf, wof, wof2.
Are there all needed ?

I understand that this framework will be used on a lot of websites in all possible webbrowser. I am just wondering if we need to handle Internet Explorer 6 with EOT.

File: paf-mvp-frontend/rollup.config.js [hunk 1]
```diff
@@ -107,8 +108,19 @@ export default [
...
+                {
+                  src: './assets/*',
+                  dest: 'dist',
+                },
+              ],
+            }),
             serve({ // dev server
               contentBase: '',
+              headers: {
+                'Access-Control-Allow-Origin': '*'
```
Comment (RomainLofaso) [inline paf-mvp-frontend/rollup.config.js:122]: Is it the same as the production configuration ?
If not, I am wondering if things can work locally but break on production.

**Comment by RomainLofaso** [comment]
I consider that my comments aren't blocking the merge. I validate your PR. If my comments require changes, do it here or in another PR. Up to you.

**Review by RomainLofaso — APPROVED** [review]

File: paf-mvp-demo-express/src/views/publisher/index.hbs [hunk 2]
```diff
@@ -2,9 +2,8 @@
 <html lang="html">
 <head>
     <!-- Prebid SSO integration -->
```
[the remark this answers is not in the document]
Comment (kovtun-d) [reply paf-mvp-demo-express/src/views/publisher/index.hbs:4]: Will do

File: paf-mvp-demo-express/src/views/publisher/index.hbs [hunk 3]
```diff
@@ -136,6 +135,9 @@
     {{/if}}
 </head>
 <body>
+<script src="https://{{cdnDomain}}/assets/app.bundle.js" data-proxy="{{proxyHostName}}"></script>
```
[the remark this answers is not in the document]
Comment (kovtun-d) [reply paf-mvp-demo-express/src/views/publisher/index.hbs:138]: Yes, I confirm. James has left a good [explanation](https://github.com/prebid/paf-mvp-implementation/pull/120/files#diff-496599c1ad87073bdb9990fa885374e3dca0e034fe0b8626e23ce4503587fe62R16)

File: paf-mvp-frontend/rollup.config.js (hunk 1 above)
Comment (kovtun-d) [reply paf-mvp-frontend/rollup.config.js:122]: dev server is for local development only. I added this header for fonts files on the prod server

**Pull request merged**